tests/routes/pro/post_stocks_test.py

```diff
@@ -0,0 +1,211 @@
+
+        response_dict = response.json
+        assert len(response_dict["stocks"]) == len(stock_data["stocks"])
+
+        created_stock = Stock.query.filter_by(id=dehumanize(response_dict["stocks"][0]["id"])).first()
+        assert offer.id == created_stock.offerId
+        assert created_stock.price == 20
+
+    def test_edit_one_stock(self, app):
+        # Given
+        offer = offers_factories.ThingOfferFactory()
+        existing_stock = offers_factories.StockFactory(offer=offer, price=10)
+        offers_factories.UserOffererFactory(
+            user__email="user@example.com",
+            offerer=offer.venue.managingOfferer,
+        )
+
+        # When
+        stock_data = {
+            "offerId": humanize(offer.id),
+            "stocks": [{"id": humanize(existing_stock.id), "price": 20}],
+        }
+        response = TestClient(app.test_client()).with_auth("user@example.com").post("/stocks/bulk/", json=stock_data)
+
+        # Then
+        assert response.status_code == 201
+
+        response_dict = response.json
+        assert len(response_dict["stocks"]) == len(stock_data["stocks"])
+
+        edited_stock = Stock.query.filter_by(id=dehumanize(response_dict["stocks"][0]["id"])).first()
+        assert edited_stock.price == 20
+
+    def test_upsert_multiple_stocks(self, app):
+        # Given
+        offer = offers_factories.ThingOfferFactory()
+        existing_stock = offers_factories.StockFactory(offer=offer, price=10)
+        offers_factories.UserOffererFactory(
+            user__email="user@example.com",
+            offerer=offer.venue.managingOfferer,
+        )
+        booking_limit_datetime = datetime(2019, 2, 14)
+
+        # When
+        stock_data = {
+            "offerId": humanize(offer.id),
+            "stocks": [
+                {
+                    "id": humanize(existing_stock.id),
+                    "price": 20,
+                    "quantity": None,
+                    "bookingLimitDatetime": serialize(booking_limit_datetime),
+                },
+                {
+                    "price": 30,
+                    "quantity": None,
+                    "bookingLimitDatetime": serialize(booking_limit_datetime),
+                },
+                {
+                    "price": 40,
+                    "quantity": None,
+                    "bookingLimitDatetime": serialize(booking_limit_datetime),
+                },
+            ],
+        }
+        response = TestClient(app.test_client()).with_auth("user@example.com").post("/stocks/bulk/", json=stock_data)
+
+        # Then
+        assert response.status_code == 201
+
+        response_dict = response.json
+        assert len(response_dict["stocks"]) == len(stock_data["stocks"])
+
+        for idx, result_stock_id in enumerate(response_dict["stocks"]):
+            expected_stock = stock_data["stocks"][idx]
+            result_stock = Stock.query.filter_by(id=dehumanize(result_stock_id["id"])).first()
```

Comment:
```suggestion
            result_stock = Stock.query.get(dehumanize(result_stock_id["id"]))
```

(Je chipote. Mais comme il n'y a qu'un seul stock avec cet id, il n'est pas nécessaire d'utiliser `first()`.)